README.md

```diff
@@ -1,2 +1,11 @@
-# agent-modules
-A repository of public agent modules
+<p align="center"><img src="assets/logo_and_name.png" alt="Grafana Agent logo"></p>
+
+[Modules](https://grafana.com/docs/agent/latest/flow/concepts/modules/) are a 
+way to create Grafana Agent [Flow](https://grafana.com/docs/agent/latest/flow/)
+configurations which can be loaded as a component. Modules are a great way to
+parameterize a configuration to create reusable pipelines.
+
+## Contents
+- <b>modules</b>: A library of usable modules out of the box
```

Comment:
I think we can axe modules folder, and example since this is a totally valid module. Linking to this from the readme as a good example to start is a good idea.